src/assets/stylesheets/_variables.scss

```diff
@@ -16,4 +21,19 @@ $body-color: $white;
 
 $font-family-sans-serif: 'Neuzeit S LT Std';
 
-$font-size-base: 1.0625rem; // 17px
+$font-size-base: map-get($font-sizes, md);
```

Comment:
If we set this here with `17px`, it seems that the other places using `rem` will be calculated base on that.
The reference here: https://www.sitepoint.com/understanding-and-using-rem-units-in-css/

So the next line about `$input-padding-y: 1rem;` means that it has `17px`, right?

Reply:
Just checked Bootstrap, Seems it does not affect the base rem.

On Bootstrap document
> No base font-size is declared on the <html>, but 16px is assumed (the browser default). font-size: 1rem is applied on the <body> for easy responsive type-scaling via media queries while respecting user preferences and ensuring a more accessible approach.

So there are no `font-size` set on the html tag. So the base rem is still 16px 👍

Reply:
Ahh, great one!